aad/spring-cloud-azure-starter-active-directory/aad-resource-server-by-filter/README.md

```diff
@@ -103,46 +103,29 @@ spring:
           client-secret: ${AZURE_CLIENT_SECRET}
         profile:
           tenant-id: ${AZURE_TENANT_ID}
-        redirect-uri-template: http://localhost:8080/
-        # groups that you created in your Azure AD tenant
         user-group:
           allowed-group-names: group1,group2
-        # Optional, the default value is 
-        # environment: global  
+        redirect-uri-template: http://localhost:8080/
```

Comment:
we don't need this I think?

Reply:
It's already configured in application.yml and checking the result needs this.